protocol/v2/p2p/testing/utils.go

```diff
@@ -0,0 +1,408 @@
+package testing
+
+import (
+	"context"
+	crand "crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/bloxapp/ssv/protocol/v2/message"
+	protocolp2p "github.com/bloxapp/ssv/protocol/v2/p2p"
```

Comment:
I think we need to decide at this point what names we use for importing packages inside `protocol`: If `protocol` is a prefix or a postfix. I used it as a postfix in a few places, but probably having it as a prefix is better, also we already have a `spec` prefix.

@lior-blox @nivBlox Your thoughts?